### PR TITLE
Added Footer

### LINKED ISSUE #24: feat: footer for the app
## Description

Add a footer for the app with @copyright to our website!

### PR DESCRIPTION
**Pull request by yashikajotwani12**
### What changes does this PR do?
 Add a Footer to the app
### Have you tested the working of the feature you fixed?(Yes/No)
Yes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "@testing-library/user-event": "^12.8.3",
     "dotenv": "^16.0.3",
     "firebase": "^8.6.3",
+    "mdb-react-ui-kit": "^4.2.0",
```
Comment (vedant-jain03) [inline package.json]: dependencies for react are getting broken!
Do not use ui-kit for now. It will broke other UI as well!

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

File: .github/workflows/build-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+name: Build & deploy
```
Comment (vedant-jain03) [inline .github/workflows/build-deploy.yml:1]: Could we have this file in other separate PR?

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

File: src/App.css [hunk 3]
```diff
@@ -85,4 +85,15 @@
...
+footer {
+  position: fixed;
+  background-color: #2c393e;
+  color: white;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  padding: 19px;
+  text-align: center;
+  font-size: 18px;
```
Comment (vedant-jain03) [inline src/App.css]:
```suggestion
    position: fixed;
    background-color: #00000000;
    color: white;
    bottom: 0;
    left: 0;
    right: 0;
    padding: 5px;
    text-align: center;
    font-size: 10px;
    border-top: 1px solid gray;
```

Try this also,

we can update the `.maincontainer` height to `84.5px`.

@yashikajotwani12 WDYT?

Comment (yashikajotwani12) [reply src/App.css]: I have updated as per your review.

**Review by vedant-jain03 — APPROVED** [review]

**Pull request merged**